tests/test_type_inference/test_initializer.py

```diff
@@ -0,0 +1,62 @@
+import astroid
+from typing import _ForwardRef
+from python_ta.transforms.type_inference_visitor import TypeInferer
+from python_ta.typecheck.base import TypeInfo, TypeFail
+from nose.tools import eq_
+
+
+def test_class_with_init():
+    program = """
+    class Foo:
+        a
```

Comment:
This is a less typical case, and should be avoided. These tests should use `pass` when possible.

Reply:
sorry just to clarify, what makes the case less typical? I intended for this to be a base case of a class definition that had an explicitly declared `__init__` function, as opposed to the later tests that didn't explicitly declare them. Or did you mean the declaration of the `a` variable?

Reply:
Yes, it is indeed the `a` which is not typical. Think about making this test as minimal as possible (hence the `pass` comment).